src/libraries/System.Data.Common/tests/System/Data/SqlTypes/SqlStringSortingTest.cs

```diff
@@ -86,7 +87,16 @@ private static void SqlStringDefaultCompareOptionTest(int localeID)
             SqlString str1;
             SqlString str2;
 
-            for (int i = 0; i < s_specialMatchingString.GetLength(0); ++i)
+            int count = s_specialMatchingString.GetLength(0);
+
+            // Some of Windows versions have a regression, so ignore last entry in the s_specialMatchingString if this is the case.
+            if (PlatformDetection.IsWindows10Version1903OrGreater &&
```

Comment:
can you use `!IsNotWindowsRegressedVersion()` here?

Reply:
No, because it is a different test.